main.c

```diff
@@ -981,10 +1001,10 @@ int main(int argc, char **argv)
 	bool new_path = false;
 	bool watching_cfg = false;
 	struct flock lock_fd = {0, };
-	char *log_dir = NULL;
 	int fd;
 	int ret;
 
+	tcmu_cfg = tcmu_initiatize_options();
```

Comment:
Please defend the possible NULL pointer, or the crash may happen, the crash will be much harder to debug .